packages/api/src/core/prepareAuth.ts

```diff
@@ -72,7 +72,7 @@ export default function prepareAuth(authKeys: (number | string)[][], operation:
           );
         }
 
-        if (scheme.in === 'query' || scheme.in === 'header') {
+        if (scheme.in === 'query' || scheme.in === 'header' || scheme.in === 'cookie') {
```

Comment:
This missing line was preventing cookie-based usages of `sdk.auth()` from working.